### DISCUSSION TITLE
Multi language docs

### DISCUSSION THREAD

**Discussion by Zac0511** [Vanilla-OS/handbook#103] [Q&A]
I saw that the docs' files where in the `en` folder, so can i create a `fr` folder in the `articles` folder for putting french documentation, and others can create other folders for other languages ?

**Comment by kbdharun** [comment]
> I saw that the docs' files where in the `en` folder, so can i create a `fr` folder in the `articles` folder for putting french documentation, and others can create other folders for other languages ?

Thanks for your interest, atm we aren't accepting new translations for docs as it is being re-written for our next version. Feel free to add missing French translations for our new components at https://hosted.weblate.org/projects/vanilla-os/-/fr/.

I will let you know when the docs/handbook is open for new translations in future.
